odl/__init__.py

```diff
@@ -43,7 +43,7 @@
 from . import deform
 from . import ufunc_ops
 from . import datasets
-
+from . import contrib
```

Comment:
Yeah, this is actually correct. It only makes `contrib` appear as a name on the top level, but it's not filled with anything.